src/Randomizer.App/ViewModels/TrackerMapLocationViewModel.cs

```diff
@@ -141,17 +159,52 @@ public ImageSource IconImage
             get
             {
                 var image = "blank.png";
-                if (ClearableLocationsCount > 0 && ClearableLocationsCount == UnclearedLocationsCount)
-                {
-                    image = "accessible.png";
-                }
-                else if (ClearableLocationsCount > 0 && ClearableLocationsCount < UnclearedLocationsCount)
+
+                if (Locations == null)
                 {
-                    image = "partial.png";
+                    var region = (IHasReward)Region;
```

Comment:
I can't tell from the diff, but can't this fail? Or is `Region` always a dungeon or SM boss area here?

Reply:
It would only fail in the case of a bad config, and I doubt anyone would be modifying the maps config file.

Reply:
So it could still conceivably come up, then. I do want to throw a better error, but for now we could probably just leave this as TODO.